Detectors/ITSMFT/ITS/tracking/include/ITStracking/StandaloneDebugger.h

```diff
@@ -15,21 +15,91 @@
 #ifndef O2_ITS_STANDALONE_DEBUGGER_H_
 #define O2_ITS_STANDALONE_DEBUGGER_H_
 
+#include <string>
+#include <iterator>
+
+// Tracker
+#include "DataFormatsITS/TrackITS.h"
+#include "ITStracking/ROframe.h"
+
 namespace o2
 {
 
-class MCCompLabel;
+// class MCCompLabel;
 
 namespace utils
 {
 class TreeStreamRedirector;
 }
 
+// namespace its
+// {
+// class TrackITSExt;
+// }
+
 namespace its
 {
 class Tracklet;
 class Line;
 class ROframe;
+class ClusterLines;
+
+using constants::its::UnusedIndex;
+
+struct FakeTrackInfo {
+ public:
+  FakeTrackInfo();
+  FakeTrackInfo(const ROframe& event, TrackITSExt& track) : isFake{false}, isAmbiguousId{false}, mainLabel{UnusedIndex, UnusedIndex, UnusedIndex, false}
+  {
+    occurrences.clear();
+    for (size_t iCluster{0}; iCluster < 7; ++iCluster) {
+      int extIndex = track.getClusterIndex(iCluster);
+      o2::MCCompLabel mcLabel = event.getClusterLabels(iCluster, extIndex);
```

Comment:
in principle, a cluster may have multiple labels. What is the reason to copy the labels into the ROframe, rather than fetching them by cluster index from the MC container?

Reply:
Well, I just fot this from the approach used in the tracker to access the labels, at some point I realised it is a duplication.
Unless I am forgetting some strong reason we might consider to simplify this, will check with Max to be sure.